benchmarks/benchmarks/stats.py

```diff
@@ -95,7 +185,6 @@ def time_distribution(self, distribution, properties):
                 stats.beta.rvs(size=1000, a=5, b=3, loc=4, scale=10)
             elif properties == 'fit':
                 stats.beta.fit(self.x, loc=4, scale=10)
-
     # Retain old benchmark results (remove this if changing the benchmark)
```

Comment:
```suggestion

    # Retain old benchmark results (remove this if changing the benchmark)
```
Let's not remove this line. Don't want to touch it at all.